api/v1beta1/virtinkcluster_types.go

```diff
@@ -22,6 +22,9 @@ type VirtinkClusterSpec struct {
 	// of type ClusterIP, which will make workload cluster only accessible within the same cluster.
 	ControlPlaneServiceType *corev1.ServiceType `json:"controlPlaneServiceType,omitempty"`
 
+	// ControlPlaneServiceAnnotations will be appended to the control plane service annotations.
```

Comment:
Maybe we should consider the possibility to let user create the service and we simply use it (or at most add labels to it)? For now we may be dealing with annotations, and someday we may find ourselves dealing with more fields of service. It could be cumbersome then.

Reply:
That's a good idea! I'm trying to make it come true.